tests/_util.py

```diff
@@ -65,3 +67,12 @@ def disable_asgi_non_coroutine_wrapping():
 
     if should_wrap:
         os.environ['FALCON_ASGI_WRAP_NON_COROUTINES'] = 'Y'
+
+
+def as_params(*values, prefix=None):
+    if not prefix:
+        prefix = ''
+    return [
+        pytest.param(*value, id=f'{prefix}_{i}' if prefix else f'{i}')
```

Comment:
I guess this currently does not work when parametrize over a single param.

Should it be fixed?

Reply:
If the whole suite passes with your fixes and `pytest-xdist`, it can stay _as-is_, just add a `NOTE`.

Reply:
OK adding